docs/UserGuide.md

```diff
@@ -318,13 +318,13 @@ search through the client list for any matches with `Amos`.
 
 Expected Output:
 ```
-find -property f/Bob The Builder
```

Comment:
Good deletion of input lines which will not be displayed in the expected output.